src/styles/blocks/card.scss

```diff
@@ -0,0 +1,113 @@
+.card {
+  width: $card-width;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  font-family: Roboto, serif;
+  font-weight: 400;
+
+  transition: all linear $main-during;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    display: flex;
+    background-image: url(../images/imac.jpeg);
+    background-size: contain;
+    background-repeat: no-repeat;
+    height: 134px;
+    margin-inline: 3px;
+    margin-bottom: 40px;
+    padding: 0;
+    background-origin: border-box;
+  }
+
+  &__text {
+    font-size: 10px;
+    line-height: 14px;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+
+    margin-bottom: 4px;
+    color: $main_accent-color;
+
+    transition: all linear $main-during;
+  }
+
+  &__text--product-code {
+    color: $secondary-color;
```

Comment:
Ensure that the SCSS variable `$secondary-color` is defined. This variable is crucial for the color styling of the product code text.